apps/api/src/database/database-reset.ts

```diff
@@ -0,0 +1,37 @@
+import DataSource from '../config/typeorm/configuration';
+import { QueryRunner } from 'typeorm';
+import { Logger } from '@nestjs/common';
+
+const logger = new Logger('DatabaseReset');
+
+DataSource.initialize()
+  .then(async () => {
+    logger.log('Data source initialized successfully');
+
+    const queryRunner: QueryRunner = DataSource.createQueryRunner();
+
+    try {
+      await queryRunner.connect();
+
+      logger.log('Dropping notifications and migrations tables...');
+      await queryRunner.dropTable('notify_notifications', true);
```

Comment:
You have to update the code too for your tables.
basically clear DB and Run migrations again from the start.
Make sure you have tested it using package json script command.

Reply:
Updated the code and tested again